packages/orchestration/test/supports.ts

```diff
@@ -38,8 +33,11 @@ export const commonSetup = async t => {
   // To test durability in unit tests, test a particular entity with `relaxDurabilityRules: false`.
   // To test durability integrating multiple vats, use a RunUtils/bootstrap test.
   const rootZone = makeHeapZone();
-  const bld = withAmountUtils(makeIssuerKit('BLD'));
-  const ist = withAmountUtils(makeIssuerKit('IST'));
+  const { mint: _b, ...bld } = withAmountUtils(makeIssuerKit('BLD'));
```

Comment:
this could use some explaining. a perhaps more legible approach would be an option to `withAmountUtils`

Reply:
The concern seems specific to `pourPayment` and `makeFakeBankManagerKit` so explaining here seems like the way to go.